src/lib/components/Filters/CheckboxDropdownFilterForm.tsx

```diff
@@ -13,23 +13,26 @@ import {
 } from '@material-ui/core';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import ChipsList from '../Chips/ChipsList';
-import { ICheckBoxFormState } from './CheckBoxFilterForm';
+import { ICheckboxFormState } from './CheckboxFilterForm';
 import { FilterTypeEnum } from '../../types';
 
 interface IFilter {
   id: string | number;
   name: string;
 }
 
-export interface ICheckBoxDropdownFilterFormProps {
-  onFormChange: (checked: ICheckBoxFormState, filterType?) => void;
+export interface ICheckboxDropdownFilterFormProps {
+  onFormChange: (
+    checked: ICheckboxFormState,
+    filterType: FilterTypeEnum,
```

Comment:
same as CheckBoxFilterForm

Reply:
Decoupled the filter components with the `FilterTypeEnum`. Now using it in the components where these filters are used, in order to have only one function to process all the filters' changes. See `MaterialsView`, for example